models/endpoints-model.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../db/connection')
+const fs = require('fs/promises')
+
+function selectEndPoints(){
```

Comment:
we don't need a model for this, we can do this functionality in the controller :)